OpenRA.Mods.Common/Traits/Render/WithSpriteBody.cs

```diff
@@ -0,0 +1,56 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System;
+using OpenRA.Graphics;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	class WithSpriteBodyInfo : UpgradableTraitInfo, ITraitInfo, Requires<RenderSpritesInfo>
+	{
+		public readonly string StartSequence = "";
+		public readonly string Sequence = "idle";
```

Comment:
Could you add descriptions to the trait and its properties?
I only know this is a looping sequence by reading the code.